tests/e2e/go.mod

```diff
@@ -0,0 +1,193 @@
+module e2e
+
+go 1.23.1
+
+toolchain go1.23.2
+
+replace github.com/imdario/mergo => github.com/imdario/mergo v0.3.16
+
+replace github.com/alessio/shellescape => github.com/alessio/shellescape v1.4.2
```

Comment:
Here too, these should replace specific versions

Reply:
I copied this one from the sec-bundle to follow the same pattern🤣

Reply:
I know, it's still wrong in security-bundle but I didn't want to block there :sweat_smile:

Reply:
I tried removing it but `go mod tidy` always replaced it there